lib/MagicPen.js

```diff
@@ -24,7 +32,12 @@ function MagicPen(options) {
     this.output = [[]];
     this.styles = Object.create(null);
     this.installedPlugins = [];
-    this._themes = {};
+    // Ready to be cloned individually:
+    this._themes = {
+        html: { styles: {} },
+        ansi: { styles: {} },
+        text: { styles: {} }
+    };
```

Comment:
Hmm, yeah, that's a bit weird. Not sure what I was thinking. It did actually work, though.

Added commits that avoid hardcoding the list of serializers and tests that themes for non-built-in serializers can be added and overridden.